### ISSUE TITLE
Using Jinja escaping on curly braces throws 206 code

### ISSUE THREAD

**Issue by bootswithdefer** [ansible/ansible-lint#439]
# Issue Type
- Bug report

# Ansible and Ansible Lint details

```
ansible-ansible 2.7.5
  config file = /home/xxx/ansible/ansible.cfg
  configured module search path = [u'/usr/share/ansible']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.5 (default, Sep 12 2018, 05:31:16) [GCC 4.8.5 20150623 (Red Hat 4.8.5-36)]

ansible-lint 4.0.0
```

- ansible installation method: pip
- ansible-lint installation method: pip

# Desired Behaviour
No error reported.  This is using the example of escaping from the Jinja docs: http://jinja.pocoo.org/docs/dev/templates/#escaping

# Actual Behaviour (Bug report only)
```
[206] Variables should have spaces before and after: {{ var_name }}
t.yml:2
- shell: docker info --format '{{ '{{' }}json .Swarm.LocalNodeState{{ '}}' }}' | tr -d '"'
```
Test file:
```
---
- hosts: localhost
  connection: local
  become: false
  gather_facts: false
  tasks:
    - shell: docker info --format '{{ '{{' }}json .Swarm.LocalNodeState{{ '}}' }}' | tr -d '"'
```
